### ISSUE TITLE
Unable to destroy cluster cleanly because of validation error

### ISSUE THREAD

**Issue by dalberto** [mumoshu/terraform-provider-eksctl#4]
First, thanks for your work on this provider, it's worked really well overall. I have encountered an issue, however, when I try to destroy a cluster via `terraform destroy`. 

I consistently get the following error message:

```sh
Error: InvalidParameter: 1 validation error(s) found.
- missing required field, DeleteTagsInput.Resources.
```

I'm running the following:

```sh
Terraform v0.13.0-beta3
+ provider registry.terraform.io/hashicorp/aws v2.70.0
+ provider registry.terraform.io/hashicorp/random v2.3.0
+ provider registry.terraform.io/hashicorp/tls v2.2.0
+ provider registry.terraform.io/mumoshu/eksctl v0.3.2
```

The cluster does in fact appear to be destroyed, but because terraform does not exit cleanly, the state is in disarray. I manually run the following to remove the cluster from terraform's state:

```sh
terraform state rm 'module.eks_cluster.eksctl_cluster.primary'
```

**Comment by mumoshu** [comment]
@dalberto Thanks for reporting and trying the provider!

I've confirmed that it was indeed failing when you tried `terraform destroy` on a cluster without `vpc_id` attribute set.

It's fixed since v0.3.3 which I've released just now.

Would you mind giving it a shot?

**Comment by dalberto** [comment]
That fixed the issue. Thanks for the quick turnaround.
